doc/newsboat.asciidoc

```diff
@@ -349,6 +349,11 @@ subscriptions that were in the "Programming" folder or one of its subfolders
 before. This means that you will lose virtually nothing of your previously
 configured structure.
 
+=== User contributed content
+
+The `contrib/` directory contains user contributed content like scripts and
```

Comment:
Sorry for being so imprecise in my wording in #475. I meant for this paragraph to be at the end of doc/chapter-firststeps.asciidoc, so people would read it right after they went through the basics, but before they learned about migrations and such. Can you please move it there? Also, no need to make it a separate section — just add it as another paragraph in that chapter.

Reply:
Fixed @ https://github.com/newsboat/newsboat/pull/1321/commits/2cbc93e1f10172e7b497126c6ae4b5aecd98723c